EIPS/eip-6860.md

```diff
@@ -0,0 +1,241 @@
+
+#### Example 3
+
+```
+web3://vitalikblog.eth:5/
+```
+
+where the contract of **vitalikblog.eth:5** is in manual mode.
+
+The protocol will find the address of **vitalikblog.eth** from ENS on chainid 5 (Goerli). Then after determing the contract is in manual mode, the protocol will call the address with "To" = **contractAddress** and "Calldata" = "0x2F" with chainid = 5. The returned data will be treated as ABI-type "(bytes)", and the decoded bytes will be returned to the frontend, with the information that the MIME type is ``text/html``.
+
+#### Example 4
+
+```
+web3://0xe4ba0e245436b737468c206ab5c8f4950597ab7f:42170/
+```
+
+where the contract "0xe4ba0e245436b737468c206ab5c8f4950597ab7f:42170" is in manual mode.
+
+After determing the contract is in manual mode, the protocol will call the address with "To" = "0xe4ba0e245436b737468c206ab5c8f4950597ab7f" and "Calldata" = "0x2F" with chainid = 42170 (Arbitrum Nova). The returned data will be treated as ABI-type "(bytes)", and the decoded bytes will be returned to the frontend, with the information that the MIME type is ``text/html``.
+
+#### Example 5
+
+```
+web3://0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48/balanceOf/vitalik.eth?returns=(uint256)
+```
+
+where the contract "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48" is in auto mode.
+
+The protocol will find the addresses of **vitalik.eth** from ENS on chainid 1 (Mainnet) and then call the method "balanceOf(address)" of the contract with the **vitalik.eth**'s address. The returned data from the call of the contract will be treated as ABI-type "(uint256)", and the decoded data will be returned to the frontend in JSON format like `[ "0x9184e72a000" ]`, with the information that the MIME type is ``application/json``.
+
+#### Example 6
+
+```
+web3://0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48/balanceOf/vitalik.eth?returns=()
+```
+
+where the contract ”0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48“ is in auto mode.
+
+The protocol will find the address of **vitalik.eth** from ENS on chainid 1 (Mainnet) and then call the method "balanceOf(address)" of the address. The returned data from the call of the contract will be treated as raw bytes and will be encoded in JSON format like `["0x000000000000000000000000000000000000000000000000000009184e72a000"]` and returned to the frontend, with the information that the MIME type is ``application/json``.
+
+### Changes versus [ERC-4804](./eip-4804.md)
```

Comment:
I think we should use EIP-4804 (also EIP over ERC) in the whole document.

Reply:
Unfortunately, if I do so, I get errors from the EIP Walidator, which then prevents merging : 
```
error[markdown-refs]: references to proposals with a `category` of `ERC` must use a prefix of `ERC`
```